orchestrator-salt/src/main/resources/salt/salt/kerberos/server.sls

```diff
@@ -0,0 +1,40 @@
+{%- from 'kerberos/settings.sls' import kerberos with context %}
+
+/etc/krb5.conf:
+  file.managed:
+    - source: salt://kerberos/config/krb5.conf
+    - template: jinja
+
+install_kerberos:
+  pkg.latest:
+    - pkgs:
+      - krb5-server
+      - krb5-libs
+      - krb5-workstation
+
+create_db:
+  cmd.run:
+    - name: /usr/sbin/kdb5_util -P {{ kerberos.master_key }} -r {{ kerberos.realm }} create -s
+    - unless: ls -la /var/kerberos/krb5kdc/principal
+    - watch:
```

Comment:
Do we need re-execute it for every package change? If not then require is better than watch